.github/workflows/minimum_python_versions.yaml

```diff
@@ -1,4 +1,4 @@
-name: Minimum Python Versions
+name: python version check
```

Comment:
While we are at it we can upgrade the Python version used in this workflow to 3.7. We had issues before but they are solved with vermin 0.9.1, see https://github.com/netromdk/vermin/issues/32 (tried it offline and works like a charm)

Reply:
done!